configs/genesis-assets/genesis.json

```diff
@@ -244,6 +248,90 @@
       ],
       "0xffffffffffffffffffffffffffffffff"
     ],
+    [
```

Comment:
What are we doing here?

(damn I hate json for not supporting comments)

Reply:
I set the initial STRK balance for pre-deployed account as it was done for ETH.